src/bot.py

```diff
@@ -363,6 +365,11 @@ async def removeLastPoppedQueue(ctx):
     await sendMessage(ctx, Admin.brokenQueue(ctx.message.author, ctx.message.author.roles), "queue")
 
 
+@client.command(name="bq", aliases=["mentionBrokenQueue"], pass_context=True)
+async def mentionBQ(ctx, *arg):
+    await ctx.send(embed=await Admin.mentionBrokenQueue(ctx.message.content, ctx.message.author.roles, *arg))
```

Comment:
You should be able to use ctx.message.mentions here which returns a list of Discord.Mentions. We use this in a couple other places I believe. I think we had the issue where it would pick up Norm in the mention list since we have to reply, right? You can filter the mentions list to exclude any members where `isBot` is true. I believe I did something like this in the branch that I played around with if you need a reference.

Reply:
The issue we ran into wasn't that it was trying to reply to itself; since we were using the reply function in discord, the first mention in the message object was the person being replied to, and there were no more mentions listed. We tested this and found it true even if you mentioned someone in your reply to Norm. That's why we ended up taking the route of figuring out what Norm sees when we have a mention in our message content and not our message object. Thus, the `split = mentions.split("<@!")` was a workaround.